### PR TITLE
AURORA4 RESULTS.md file

### PR DESCRIPTION
**Pull request by 27jiangziyan**
- Change RESULTS to RESULTS.md
-  Update 2 new models(But still the default is the best)
Action items:
- [x]  Put the model in the google drive and update link here.

### CODE REVIEW

File: egs/aurora4/asr1/RESULTS.md [hunk 1]
```diff
@@ -1,4 +1,95 @@
...
+  - Environments (obtained by `$ get_sys_info.sh`)
+    - date: `Thu Jun 13 01:29:55 EDT 2019`
+    - system information: `Linux a09 4.9.0-4-amd64 #1 SMP Debian 4.9.65-3+deb9u1 (2017-12-23) x86_64 GNU/Linux`
+    - python version: `Python 3.6.8 :: Anaconda, Inc.
+    - espnet version: `espnet 0.3.1`
+    - chainer version: `chainer 5.0.0`
+    - pytorch version: `pytorch 1.0.0`
+    - Git hash: `04b9e1a71b3272cdf6502e40679ab17f3f3f22f6`
+  - Model files (archived to model.tar.gz by `$ pack_model.sh`)
+    - model link:
```
Comment (sw005320) [inline egs/aurora4/asr1/RESULTS.md]: Can you update a model to the Google drive I sent an invitation and put the model link here?

File: egs/aurora4/asr1/RESULTS.md [hunk 2]
```diff
@@ -7,19 +98,10 @@ exp/train_mix_pytorch_mix_shallow_wide/decode_dev_0330_decode_wordlm/result.wrd_
...
-exp/train_mix_pytorch_transformer_100epoch/decode_dev_0330_decode_lm_word65000/result.wrd_B.txt:|  SPKR     |  # Snt     # Wrd   |  Corr       Sub       Del       Ins       Err     S.Err   |
-exp/train_mix_pytorch_transformer_100epoch/decode_dev_0330_decode_lm_word65000/result.wrd_B.txt:|  Sum/Avg  |  1980      32802   |  92.4       6.4       1.2       1.0       8.6      53.3   |
-exp/train_mix_pytorch_transformer_100epoch/decode_dev_0330_decode_lm_word65000/result.wrd_C.txt:|  SPKR     |  # Snt    # Wrd   |  Corr        Sub       Del       Ins       Err     S.Err   |
-exp/train_mix_pytorch_transformer_100epoch/decode_dev_0330_decode_lm_word65000/result.wrd_C.txt:|  Sum/Avg  |   330      5467   |  93.9        5.3       0.7       0.7       6.7      43.6   |
-exp/train_mix_pytorch_transformer_100epoch/decode_dev_0330_decode_lm_word65000/result.wrd_D.txt:|  SPKR     |  # Snt     # Wrd   |  Corr       Sub       Del       Ins       Err     S.Err   |
-exp/train_mix_pytorch_transformer_100epoch/decode_dev_0330_decode_lm_word65000/result.wrd_D.txt:|  Sum/Avg  |  1980      32802   |  85.7      12.2       2.1       2.0      16.3      65.2   |
+```
 
 # First result
```
Comment (sw005320) [inline egs/aurora4/asr1/RESULTS.md]: You should change the order of the result from new to old. The first result must be put in the last.

**Comment by 27jiangziyan** [comment]
Very sorry for the late reply. There is something wrong with my computer system these two days. I have rearranged the RESULT.md and put models in the link.

**Comment by sw005320** [comment]
Thanks!

**Pull request merged**